src/components/BottomBar/WWTPanel.jsx

```diff
@@ -0,0 +1,314 @@
+    let currentPopoverHeight = this.state.currentPopoverHeight - selectionButtonsAndSearchHeight;
+
+    let skybrowserTabs = <SkybrowserTabs
+      api = {api}
+      skybrowserApi = {api.skybrowser}
+      cameraInSolarSystem = {this.state.cameraInSolarSystem}
+      targets={this.state.targetData}
+      selectedTarget={this.state.selectedTarget}
+      selectedBrowser={this.state.selectedBrowser}
+      isUsingRae = {this.state.isUsingRae}
+      isFacingCamera = {this.state.isFacingCamera}
+      currentPopoverHeight={currentPopoverHeight}
+      setCurrentTabHeight= {this.setCurrentTabHeight}
+      data={thisTabsImages}
+      selectImage = {this.selectImage}
+      currentTargetColor = {this.getCurrentTargetColor}
+      />;
+
+    const skyBrowser = <PopoverSkybrowser
+                          title="AAS WorldWide Telescope"
+                          closeCallback={this.togglePopover}
+                          detachable
+                          heightCallback={this.setCurrentPopoverHeight}
+                          >
+                          <div className={styles.row}>
+                          <Picker
+                            className={`${styles.picker} ${this.state.showOnlyNearest ? styles.unselected: styles.selected}`}
+                            onClick={() => this.setState({ showOnlyNearest: false })}>
+                              <span>All images</span> {/*<MaterialIcon className={styles.photoIcon} icon="list_alt" />*/}
+                          </Picker>
+                          <Picker
+                            className={`${styles.picker} ${this.state.showOnlyNearest ? styles.selected : styles.unselected}`}
+                            onClick={() => this.setState({ showOnlyNearest: true })}>
+                              <span>Images within view</span> {/*<MaterialIcon className={styles.photoIcon} icon="my_location" />*/}
+                          </Picker>
+                          </div>
+                          <div className={PopoverSkybrowser.styles.content}>
+                            <div className={PopoverSkybrowser.styles.scrollArea}
+                            style={{height: 'calc(100% - ' + (this.state.currentTabHeight) + 'px)'}}>
+                            {filterList}
+                            </div>
+                            {skybrowserTabs}
+                          </div>
+                          </PopoverSkybrowser>
+
+    const errorMessage =  <PopoverSkybrowser
+                          title="AAS WorldWide Telescope"
+                          closeCallback={this.togglePopover}
+                          detachable
+                          heightCallback={this.setCurrentPopoverHeight}
+                          >
+                          <CenteredLabel>
+                          The camera has to be within the solar system for the sky browser to work.
```

Comment:
indentation

But actually, I'd still recommend to just send ESLint and prettier on the code to format all these things in one click.
So I'll stop commenting on them now